test/e2e-image/overlay/kustomization.yaml

```diff
@@ -7,6 +7,12 @@ configMapGenerator:
   behavior: merge
   literals:
   - worker-processes=1
+- name: auth-headers
```

Comment:
Please don't change this file. Use the same approach used here https://github.com/kubernetes/ingress-nginx/blob/c85450c1e7eff7df36e06dc61caa51a4e2c60526/test/e2e/settings/listen_nondefault_ports.go#L37
This narrows the scope of the changes https://github.com/kubernetes/ingress-nginx/tree/master/test/e2e-image/namespace-overlays